tests/e2e/constants/kubernetes.go

```diff
@@ -0,0 +1,214 @@
+
+	PassorwdFilePath          = "/etc/vmware/wcp/.storageUser"
+	PodContainerCreatingState = "ContainerCreating"
+
+	PsodTime                     = "120"
+	PvcHealthAnnotation          = "volumehealth.storage.kubernetes.io/health"
+	PvcHealthTimestampAnnotation = "volumehealth.storage.kubernetes.io/health-timestamp"
+	ProvisionerContainerName     = "csi-provisioner"
+	QuotaName                    = "cns-test-quota"
+	RegionKey                    = "topology.csi.vmware.com/k8s-region"
+
+	RestartOperation         = "restart"
+	RqLimit                  = "200Gi"
+	RqLimitScaleTest         = "900Gi"
+	RootUser                 = "root"
+	DefaultrqLimit           = "20Gi"
+	RqStorageType            = ".storageclass.storage.k8s.io/requests.storage"
+	ResizerContainerName     = "csi-resizer"
+	ScParamDatastoreURL      = "DatastoreURL"
+	ScParamFsType            = "csi.storage.k8s.io/fstype"
+	ScParamStoragePolicyID   = "storagePolicyID"
+	ScParamStoragePolicyName = "StoragePolicyName"
+
+	Snapshotapigroup = "snapshot.storage.k8s.io"
+
+	SpsServiceName           = "sps"
+	SnapshotterContainerName = "csi-snapshotter"
+	SshdPort                 = "22"
+	SshSecretName            = "SSH_SECRET_NAME"
+	SvcRunningMessage        = "Running"
+
+	StartOperation    = "start"
+	SvcStoppedMessage = "Stopped"
+	StopOperation     = "stop"
+	StatusOperation   = "status"
+
+	SvClusterDistribution = "SupervisorCluster"
+
+	TkgClusterDistribution                    = "TKGService"
+	VanillaClusterDistribution                = "CSI-Vanilla"
+	VanillaClusterDistributionWithSpecialChar = "CSI-\tVanilla-#Test"
+	VcClusterAPI                              = "/api/vcenter/namespace-management/clusters"
+	VcRestSessionIdHeaderName                 = "vmware-api-session-Id"
+	VpxdServiceName                           = "vpxd"
+
+	VSphereCSIControllerPodNamePrefix = "vsphere-csi-controller"
+	VmUUIDLabel                       = "vmware-system-vm-uuid"
+
+	VsphereCloudProviderConfiguration = "vsphere-cloud-provider.conf"
+	VsphereControllerManager          = "vmware-system-tkg-controller-manager"
+	VSphereCSIConf                    = "csi-vsphere.conf"
+	VsphereTKGSystemNamespace         = "svc-tkg-domain-c10"
+
+	WcpServiceName = "wcp"
+	VmcWcpHost     = "10.2.224.24" //This is the LB IP of VMC WCP and its constant
+	DevopsTKG      = "test-cluster-e2e-script"
+	CloudadminTKG  = "test-cluster-e2e-script-1"
+	VmOperatorAPI  = "/apis/vmoperator.vmware.com/v1alpha1/"
+	DevopsUser     = "testuser"
+	ZoneKey        = "topology.csi.vmware.com/k8s-zone"
+	TkgAPI         = "/apis/run.tanzu.vmware.com/v1alpha3/namespaces" +
+		"/test-gc-e2e-demo-ns/tanzukubernetesclusters/"
+	Topologykey                                = "topology.csi.vmware.com"
+	EnvTopologyMap                             = "TOPOLOGY_MAP"
+	TopologyHaMap                              = "TOPOLOGY_HA_MAP"
+	TopologyFeature                            = "TOPOLOGY_FEATURE"
+	TopologyTkgHaName                          = "tkgs_ha"
```

Comment:
EnvTopologyMap , TopologyHaMap, TopologyFeature and TopologyTkgHaName -> can go to env_constants.go